src/hs/Vehicle/Backend/LossFunction/Compile.hs

```diff
@@ -35,17 +36,18 @@ instance ToJSON Domain where
   toEncoding = genericToEncoding defaultOptions
 
 data LExpr
-  = Neg LExpr
-  | Con Double
+  = Negation LExpr
+  | Constant Double
   | Min LExpr LExpr
   | Max LExpr LExpr
-  | Sub LExpr LExpr
-  | Ind LExpr LExpr
+  | Substraction LExpr LExpr
+  | IndicatorFunction LExpr LExpr
   | Var V.DBIndex
```

Comment:
As we're being verbose, can we have `Variable`, `NetworkApplication`, `Quantifier` and `TensorLiteral` as well?

Reply:
Done in the new commit